### PR TITLE
[15.0][FIX] Fix for attachment viewer visible on home page

### PR DESCRIPTION
**Pull request by bvanjeelharia**
This PR should fix the error mentioned in https://github.com/OCA/web/issues/2416

### CODE REVIEW

**Comment by bvanjeelharia** [comment]
Added fix to close the attachment viewer automatically when moving to other app

File: web_responsive/static/src/components/apps_menu/apps_menu.esm.js [hunk 1]
```diff
@@ -34,6 +34,12 @@ export class AppsMenu extends Component {
         this.state = useState({open: false});
         useBus(this.env.bus, "ACTION_MANAGER:UI-UPDATED", () => {
             this.setState(false);
+            var attachmentViewerCloseButton = document.querySelector(
```
Comment (tarteo) [inline web_responsive/static/src/components/apps_menu/apps_menu.esm.js]: In other code it uses: `this.messaging` to access mail module related components. But I don't know how to get to there maybe with `registerMessagingComponent` so this solution is fine

**Review by tarteo — APPROVED** [review]

File: web_responsive/static/src/components/apps_menu/apps_menu.esm.js (hunk 1 above)
Comment (bvanjeelharia) [reply web_responsive/static/src/components/apps_menu/apps_menu.esm.js]: Hey @tarteo , made some updates based on your suggestion!!

**Comment by pedrobaeza** [comment]
Great, it's working as expected, thanks. Please rename the commit following guidelines (https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#71commit-message):

```
[FIX] web_responsive: Hide document viewer on App switcher
```

**Comment by bvanjeelharia** [comment]
> Great, it's working as expected, thanks. Please rename the commit following guidelines (https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#71commit-message):
> 
> ```
> [FIX] web_responsive: Hide document viewer on App switcher
> ```

Updated commit msg as requested

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge patch

**Pull request merged**